### ISSUE TITLE
Readme missing

### ISSUE THREAD

**Issue by Dario55** [SainsburyWellcomeCentre/aeon_feeder#2]
I was looking at the folders, but it is a bit hard to parse because there is no explanation of what is what. Could you please add a short readme/help? For example: which are the files for 3D-printing/laser cut, which are the files for 3D rendering only? Should one open main or the single files?
thank you

**Comment by gmcphillips** [comment]
Done - more details added that  need to be merged/ pulled into the main branch
